aries-site/src/pages/templates/drill-down-navigation.mdx

```diff
@@ -1,3 +1,50 @@
-import { ComingSoon } from '../../components';
+import { DiagramPreview, Example } from '../../layouts';
+import { DrillDownCardExample, DrillDownDataTableExample, DrillDownListExample } from '../../examples';
 
-<ComingSoon />
\ No newline at end of file
+## When to Use:
+- When users need to focus on one task at a time.
+- To give information overview with the ability to drill down into specific information.
+- When screen real estate is limited.
+- When designing for many levels of functionality.
+- To divide meaningful content into sections.
+
+## Guidelines:
+- When the user is moving forward step-by-step to access further information.
+- When going from your Account > Dashboard > Service.
+- To allow users to dive deeper into related content.
+- Can be used as an individual element or in groups of related content.
+- Each element should have it’s own single content.
+
+## Drill Down Navigation Examples:
+
+### Card Example
+A visual representation of interactive and organized content.
+
+<Example caption="A diagram illustrating drill down navigation within Cards." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/drill-down-card-example.svg", light: '/templateImages/drill-down-card-example-invert.svg'}} alt="A diagram illustrating drill down navigation within cards." />
+</Example>
+
+<Example plain>
+<DrillDownCardExample />
+</Example>
+
+### List Example
+Lists provide access points that present a collection of organized and related content.
+<Example caption="A diagram illustrating drill down navigation within Lists" plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/drill-down-list-example.svg", light: '/templateImages/drill-down-list-example-invert.svg'}} alt="A diagram illustrating drill down navigation within lists." />
+</Example>
+
+<Example plain>
+<DrillDownListExample />
+</Example>
+
+### DataTable Example
+Datatables present data in a column and row format. A user can be taken to a spoke when an entire table row is clickable or if an individual cell is clickable to drill down into more information.
+
+<Example caption="A diagram illustrating drill down navigation within a DataTable." plain pad="small" previewWidth="large" height={{ min: 'small' }}>
+<DiagramPreview src={{dark: "/templateImages/drill-down-datatable-example.svg", light: '/templateImages/drill-down-datatable-example-invert.svg'}} alt="A diagram illustrating drill down navigation within a DataTable." />
```

Comment:
This diagram does not make sense to me. All rows of a DataTable should contain the same type of information per row. Having one row navigating to a spoke and another to a hub seems inconsistent. I would think a DataTable would be presenting a collection of record summaries and the navigation from the row would take me to a record detail page.

Reply:
I can see why this can be confusing. However, while the image may seem as though its implying that one row would take you to a spoke and another would take you to a hub, what the image is actually trying to explain is that rows within a table can take you to spokes but they can also take you to hubs. Given that your designs are consistent of course. 

While I do agree with you that its common for a row to take you to a detail page, we have use cases where DataTable rows take you to other hubs.